Controllers/ArmoursController.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Monster_Builder_Web_API.Services;
+using Monster_Builder_Web_API.Models.DTOs;
+using Monster_Builder_Web_API.Models.Exceptions;
+
+namespace Monster_Builder_Web_API.Controllers;
+
+[Route("api/[controller]")]
+public class ArmoursController : ControllerBase
+{
+    private readonly IArmourService _armourService;
+    public ArmoursController(IArmourService armourService) {
+        _armourService = armourService;
+    }
+
+    [HttpGet("{name}")]
```

Comment:
Rest URI names should follow a fairly strict convention. see https://restfulapi.net/resource-naming/
But even if we are not creating a rest api, URLs like api/armours/getarmours seem a little verbose and are already implied by the Http verb used